### PR TITLE
Feature/new biter

### PR DESCRIPTION
**Pull request by raulgonzalezupc**
New biter + new effects

- Warning alert for charge
- warning alert for bite
- warning alert for explosion
- passive effects
- explosion effects

### CODE REVIEW

File: Scripts/src/Script/BiterStateAttack.cpp [hunk 1]
```diff
@@ -32,9 +33,14 @@ void BiterStateAttack::OnStateUpdate()
 
 	float current_percentage = enemy->animation->GetCurrentClipPercentatge();
 	float3 attack_center = enemy->current_position + (enemy->current_velocity.Normalized() * enemy->bite_distance) + enemy->offset;
-
+	
+	if (current_percentage >= init_attack_percentage && !enemy->warning_basic_attack_vfx->IsPlaying())
+	{
+		//enemy->warning_basic_attack_vfx->Play();
```
Comment (Dartemiss) [inline Scripts/src/Script/BiterStateAttack.cpp:39]: Why is this commented?

File: Scripts/src/Script/BiterStateAttack.cpp [hunk 2]
```diff
@@ -32,9 +33,14 @@ void BiterStateAttack::OnStateUpdate()
...
 	float3 attack_center = enemy->current_position + (enemy->current_velocity.Normalized() * enemy->bite_distance) + enemy->offset;
-
+	
+	if (current_percentage >= init_attack_percentage && !enemy->warning_basic_attack_vfx->IsPlaying())
+	{
+		//enemy->warning_basic_attack_vfx->Play();
+	}
 	if (current_percentage >= init_attack_percentage && current_percentage <= final_attack_percentage && !attacked)
 	{
+		//enemy->warning_basic_attack_vfx->Disable();
```
Comment (Dartemiss) [inline Scripts/src/Script/BiterStateAttack.cpp:43]: Why is this commented?

**Review by Dartemiss — APPROVED** [review]
GJ! Biter looks so cool now! :D

**Review by mariofv — APPROVED** [review]
Looks amazing, GJ

**Pull request merged**